src/set_theory/ordinal/arithmetic.lean

```diff
@@ -882,11 +882,12 @@ bfamily_of_family'_typein  _ f i
 
 @[simp] theorem family_of_bfamily'_enum {ι : Type u} (r : ι → ι → Prop) [is_well_order ι r] {o}
   (ho : type r = o) (f : Π a < o, α) (i hi) :
-  family_of_bfamily' r ho f (enum r i (by rwa ho)) = f i hi :=
+  family_of_bfamily' r ho f (enum r ⟨i, by rwa ho⟩) = f i hi :=
 by simp only [family_of_bfamily', typein_enum]
 
-@[simp] theorem family_of_bfamily_enum (o : ordinal) (f : Π a < o, α) (i hi) :
-  family_of_bfamily o f (enum (<) i (by { convert hi, exact type_lt _ })) = f i hi :=
+@[simp] theorem family_of_bfamily_enum (o : ordinal) (f : Π a < o, α) (i) (hi : i < o)
+  (h := hi.trans_eq (type_lt _).symm) :
```

Comment:
We need this `opt_param` to rewrite in both directions.

Reply:
No you don't, because proofs are defeq.